internal/pkg/testhelpers/core.go

```diff
@@ -117,25 +58,13 @@ func RequireNewFakeActor(t *testing.T, vms vm.Storage, addr address.Address, cod
 // RequireNewFakeActorWithTokens instantiates and returns a new fake actor and requires
 // that its steps succeed.
 func RequireNewFakeActorWithTokens(t *testing.T, vms vm.Storage, addr address.Address, codeCid cid.Cid, amt types.AttoFIL) *actor.Actor {
-	// act := actor.NewActor(codeCid, amt)
-	// var err error
-	// rawHead, err := (&actor.FakeActor{}).InitializeState(vms, &actor.FakeActorStorage{})
-	// require.NoError(t, err)
-	// act.Head = e.NewCid(rawHead)
-	// require.NoError(t, vms.Flush())
-	// return act
+
```

Comment:
I doubt these methods are being referenced. They should be deleted.

Reply:
I hear you but they're used in skipped tests I'd rather leave around to be reinstated.